lib/versioning/docker/index.spec.ts

```diff
@@ -131,7 +131,7 @@ describe('versioning/docker/index', () => {
         '3.8.0',
       ];
 
-      expect(versions.sort(docker.sortVersions)).toEqual([
+      expect(versions.sort(docker.sortVersions.bind(docker))).toEqual([
```

Comment:
```suggestion
      expect(versions.sort((a,b) => docker.sortVersions(a,b))).toEqual([
```
Don't like `bind`, removes any type safety